web/src/components/CreateTestSteps/CreateTestStepFooter.tsx

```diff
@@ -10,28 +11,42 @@ interface IProps {
 const CreateStepFooter = ({isValid, onNext}: IProps) => {
   const {stepNumber, stepList, onPrev, isLoading} = useCreateTest();
 
+  const navigateFunction = useNavigate();
   return (
     <S.Footer>
-      {Boolean(stepNumber) && (
+      {stepNumber > 0 ? (
         <Button data-cy="create-test-prev-button" type="primary" ghost onClick={onPrev}>
           Previous
         </Button>
-      )}
-      {stepNumber < stepList.length - 1 ? (
-        <Button data-cy="create-test-next-button" disabled={!isValid} onClick={onNext} type="primary">
-          Next
-        </Button>
       ) : (
+        <div />
```

Comment:
This empty div looks weird, is there another way we can achieve the same functionality without having it?

Reply:
fixed